Source/WTF/wtf/WTFConfig.cpp

```diff
@@ -73,13 +73,19 @@ void setPermissionsOfConfigPage()
     std::call_once(onceFlag, [] {
         mach_vm_address_t addr = std::bit_cast<uintptr_t>(static_cast<void*>(WebConfig::g_config));
         auto flags = VM_FLAGS_FIXED | VM_FLAGS_OVERWRITE | VM_FLAGS_PERMANENT;
-        auto prot = VM_PROT_READ | VM_PROT_WRITE;
 
         auto attemptVMMapping = [&] {
-            // We use remap instead of map here to avoid zeroing the config,
-            // since WebKit components like bmalloc can potentially use the
-            // config prior to WTF initialization.
-            return mach_vm_remap(mach_task_self(), &addr, ConfigSizeToProtect, pageSize() - 1, flags, mach_task_self(), addr, false, &prot, &prot, VM_INHERIT_DEFAULT);
```

Comment:
Is there a reason to not try remap first then fallback to this?

Reply:
That should be fine even if they don't have remap in the sandbox.

Reply:
That's possible, but now it makes the code work 2 way and makes it fallout only happen on one platform, and therefore is harder to test for.  It's not a big deal, so I'm ok with either solution.